pkgs/by-name/do/dokieli/package.nix

```
@@ -10,27 +10,28 @@
 
 mkYarnPackage rec {
```

Migrating from `mkYarnPackage` to `stdenv.mkDerivation`

```suggestion
stdenv.mkDerivation (finalAttrs: {
```